new/LIT7097SanaSom.xml

```diff
@@ -0,0 +1,103 @@
+                        The composition has no author attribution in the manuscript. In fact, the work appears to be composed of two
+                        different <foreign xml:lang="gez">Gubāʾa Qānā</foreign>-type <foreign xml:lang="gez">qəne</foreign>s
+                        and one spurious verse. The first <foreign xml:lang="gez">qəne</foreign> 
+                        (rhyme: <foreign xml:lang="gez">ው</foreign>) 
+                        is made up of verses 5–7, which
+                        should be reanalyzed as two verses by uniting verses 1–5 and verses 6–7, respectively. The second 
+                        <foreign xml:lang="gez">qəne</foreign>
+                        (rhyme: <foreign xml:lang="gez">ት</foreign>), which also has a separate circulation in a printed book
+                        (<bibl><ptr target="bm:Afawarq1984Yageezqeneyat"></ptr><citedRange unit="page">230-231, no. 246</citedRange></bibl>), 
+                        is made up of verses 9–12, which should be reanalyzed as two verses by coalescing verses 10–12 into one verse.
+                        Verse 8 is spurious.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Poetry"/>
+                    <term key="Qene"/>
+                    <term key="GubaeQana"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="JK" when="2024-10-11">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT7097SanaSom" passive="UppQ1524o"/>
+                    <relation name="ecrm:P129_is_about" active="LIT7102QeneYom" passive="PRS3430DawitII"/>
+                    <relation name="saws:contains" active="LIT7097SanaSom" passive="LIT7102QeneYom"/>
+                </listRelation>
+            </div>
+            <div type="edition" xml:lang="gez">
+                <note>This edition is based on <ref type="mss" corresp="UppQ1524o"/> on fols 4r, 5r.
+                    Division into verses is indicated with ፤ .
+                    The end of the poem, as well as some poem-internal verse divisions, are indicated with ። .
+                    Verse 5 is indented.
+                </note>
+                <div type="textpart" xml:id="p1">
+                    <l n="1">ጸንአ፡ ጾም፡ <sic resp="JK">ነግረ፡</sic> ኢሳይያ፡ አብ፤</l>
+                    <l n="2">አምጣነ፡ ባቲ፡ ወአሉ፤</l>
+                    <l n="3">እስከ፡ ፍና፤</l>
+                    <l n="4">ሰርክ፡ ሕዝብ፤</l>
+                    <l n="5">መፍትው።</l>
```

Comment:
I understand it that p1 is analyzed by @DenisNosnitsin1970 as containing two lines (currently 1-5 and 6-7); the punctuation mark ፤ is just a semicolon and not a line divider

Reply:
so we have 
`poem1 `with (it is OK to use whichever xml:id but usually we think `p` is part in manuscripts)
line 1 (now 1-5)
line 2 (now 6-7)

Reply:
You suggest to give the edition based on our interpretation of the data, rather than what is found in the manuscript. Granted that we reanalyse the entire composition as strange mix of two qenes plus one extra line, I think it's better to keep the line analysis of the manuscript and describe our interpretation in words (as I did). Is this ok?

Reply:
I suggest that the text edition (in LIT records) is about text, and transcription (in Manuscripts) follows the manuscripts; in this case Denis's view is also confirmed by the printed edition of one of the parts

Reply:
As I understand it, the manuscript attests to a composite text to which it is the only witness. It may be that this composite text probably originated in a misunderstanding of the sources (= the copyist merged at least two different qene into one text), but now it is nevertheless _one_ text (according to layout in the manuscript, numbering, etc.). This text contains another text, which is the qene identified by Denis and found in the printed book. But the rest of the composite text has no attestation outside of this context, as far as we are aware.

LIT7097SanaSom is for the composite text. This text has no unifying rhyme and I don't think it makes sense to try to trace its history in the edition. I would say it's better to present the text as it is transmitted.